misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-dialog.less

```diff
@@ -0,0 +1,132 @@
+.modal-backdrop {
+    z-index: -1;
+}
+.bootstrap-dialog {
+    /*** Icon animation* Copied from font-awesome: http://fontawesome.io;
```

Comment:
this comment seems to be never closed

Reply:
Does not seem to affect anything oddly enough, But I'll correct it.